projects/arch-ethereum-web-3/index.js

```diff
@@ -0,0 +1,79 @@
+const sdk = require("@defillama/sdk");
```

Comment:
Can you change the current file to below code and delete the other two files? Instead of counting token supply * price, I am counting the tokens backing the index

    const { sumTokens } = require('../helper/unwrapLPs');
    const sdk = require('@defillama/sdk')

    const contract = '0xe8e8486228753E01Dbc222dA262Aa706Bd67e601'
    const abi = {
      "inputs": [],
      "name": "getComponents",
      "outputs": [
        {
          "internalType": "address[]",
          "name": "",
          "type": "address[]"
        }
      ],
      "stateMutability": "view",
      "type": "function"
    }

    async function tvl(timestamp, block) {
      const { output: tokens } = await sdk.api.abi.call({ target: contract, abi, block })
      const toa = tokens.map(t => [t, contract])
      return sumTokens({}, toa, block)
    }

    module.exports = {
      ethereum: {
        tvl,
      },
    };